### DISCUSSION TITLE
Discord.js v13 Handler v10.0.0

### DISCUSSION THREAD

**Discussion by MGalaCyber** [MGalaCyber/Discord.js-v13-Command-Handler-Template#9] [Announcements]
➕ add .gitignore files
➕ add new client files
➕ add new folder & files functions
➕ add new cooldown on Message & Slash Commands
➕ add new Permissions Nsfw Only on Message & Slash Commands
➕ add new permissions Bot/User on Message & Slash Commands
➖ remove Package-lock.json files
🛠 update Packages
🛠 update Folder name
🛠 update Structures folder files
🛠 update Structures script
🛠 update Slash Commands script code
🛠 update script index.js
👾 Fix antiCrash handler
👾 Fix Events handler (pull.bind error)
👾 Fix bug

**Full Changelog**: https://github.com/MGalaCyber/Discord.js-v13-Handler-Template/compare/v8.5.0...v10.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/MGalaCyber/Discord.js-v13-Handler-Template/releases/tag/v10.0.0'>Discord.js v13 Handler v10.0.0</a>.</em>

**Comment by SrHell** [comment]
why is it giving this error
(node:3468) DeprecationWarning: The message sweeping client options are deprecated, use the global sweepers instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
[WARNING] DeprecationWarning: The message sweeping client options are deprecated, use the global sweepers instead.

**Reply by MGalaCyber** [reply]
- It's not an error, it's just logs from one of the NODEJS systems, usually it will show up if you use nodejs v16^.

- And it will still show up due to some structure of `discord.js v13^`, like the module part (const client = new Client()) if I'm not mistaken when I created this handler.

- And also the effect of some changes to `MessageCreate Event`, and reportedly message commands will be replaced with slash commands

- So it has no effect on the performance of the handler,

- You can also find out about it on forums like **_Stackoverflow_**, **_Github_**, **_Discord.js_** and **_Nodejs_**
